saffron/src/blob.rs

```diff
@@ -92,28 +92,102 @@ impl<G: CommitmentCurve> FieldBlob<G> {
     }
 }
 
+#[cfg(test)]
+mod blob_test_utils {
+    use proptest::prelude::*;
+
+    #[derive(Debug)]
+    pub struct BlobData(pub Vec<u8>);
+
+    #[derive(Clone, Debug)]
+    pub enum DataSize {
+        Small,
+        Medium,
+        Large,
+    }
+
+    // TODO: This is fine for now, but figure out what the realistic expectations are.
+    impl DataSize {
+        fn size_range_bytes(&self) -> (usize, usize) {
+            let c = 1_000_000;
+            match self {
+                Self::Small => (1, 2 * c),
+                Self::Medium => (5 * c, 10 * c),
+                Self::Large => (50 * c, 75 * c),
```

Comment:
I'm a little worried about the spaces in between where we don't test.  Should I be?

Reply:
Yeah this is pretty ad-hoc, what do you think about https://github.com/o1-labs/proof-systems/pull/2970/commits/b815f49d3efdd5424907e7b55ed5c319137fd664 ? Or any other ideas?